src/components/Game.js

```diff
@@ -0,0 +1,66 @@
+import { useMemo, useState } from "react";
+import Board from "./Board";
+
+function Game(props) {
+  const resetbox = () => {
+    return Array(9).fill("");
+  };
+  const [xIsNext, setxIsNext] = useState(true);
+  const [box, setBox] = useState(resetbox);
+
+  const handleClick = (i) => {
+    if (calculateWinner(box) || box[i]) {
+      return;
+    }
+    box[i] = xIsNext ? "X" : "O";
+    setBox(box);
+    setxIsNext(!xIsNext);
+  };
+
+  const winner = useMemo(() => {
+    let turn;
+    if (calculateWinner(box)) {
+      return (turn = calculateWinner(box) + " is Winner!");
```

Comment:
as you would return `turn` anyway, might just return without reassign `turn` variable